types/Address/Address.d.ts

```diff
@@ -0,0 +1,142 @@
+import { IObjectWithId } from '../base';
+import { DeepPartial } from '../utils';
+import { IAddressCreateParameters } from './AddressCreateParameters';
+import { IVerifications } from './Verifications';
+
+/**
+ * Address objects are used to represent people, places, and organizations in a number of contexts.
+ * For example, a Shipment requires a to_address and from_address to accurately calculate rates and generate postage.
+ *
+ * Additionally, EasyPost offers several verification tools that can be used to detect deliverability issues, correct minor errors in spelling/formatting, and determine if an Address is residential or not (which has a significant effect on Shipment rating for many carriers).
+ *
+ * @see https://www.easypost.com/docs/api/node#address-object
+ */
+export declare interface IAddress extends IObjectWithId<'Address'> {
```

Comment:
What does prepending `I` to all of these classes do? Is this to signify it's an interface?